### ISSUE TITLE
EventID 4625 metadata doesn't match

### ISSUE THREAD

**Issue by JPvRiel** [JPvRiel/wef-reference#2]
According to https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4625, the Description should read:

> An account failed to log on

However, what is looked up is in fact "An account was successfully logged on." which belongs to event ID 4624.

There may be a looping/off by one error that caused this?

To confirm it's wrong, the powershell data was correct:

```PowerShell
$m = Get-WinEvent -ListProvider 'Microsoft-Windows-Security-Auditing'
$m.Events | ?{ $_.Id -eq 4625 } | Select Id, Version, Description
$m.Events | ?{ $_.Id -eq 4625 } | Select Description | Format-List
```

It produced:

```
Description : An account failed to log on.
```

Vs the the following output to: query_combinations.yml

```yaml
    - Id: 4625
      Metadata:
        Description: An account was successfully logged on.
```

**Comment by JPvRiel** [comment]
Serious bug fixed by

**Comment by JPvRiel** [comment]
Closed by 7a742afbea97dabec62abb9640fc482c6c9ad082
